workbench/test/test_views.py

```diff
@@ -101,8 +102,12 @@ def test_xblock_with_handler():
 
     # Initially, the data is the default.
     response = client.get("/view/testit/")
-    assert_true("The data: u'def'." in response.content)
-    parsed = response.content.split(':::')
+    response_content = response.content.decode('utf-8')
+    assert_true(
+        "The data: u'def'." in response_content
+        or "The data: 'def'." in response_content
+    )
+    parsed = response_content.split(':::')
```

Comment:
Has this assert been expanded to cover differences in the response depending upon python versions?

Reply:
Should this split be updated similarly to what you have done here: https://github.com/edx/xblock-sdk/pull/156/files#diff-23557a57a78b11a8a67ffe3b0e379857R227

Reply:
I'm not sure I understand what you're asking, could you phrase it differently?

I made changes to the `assert_true` statement to handle the different python versions, as you can see – depending on which version you run the test with you'll either get `u'def'` or `'def'` in the `response_content`.

There's no need to `decode` the `split` since `response_content` has already been decoded by that time, but maybe you're suggesting that I use `response.content.decode('utf-8').split(':::')` anyway so it's more standardised throughout the test file?

Reply:
The "or" in the assertion troubles me, because it could allow an incorrect response, since it isn't tied to the Python version.  This might work better:
```
expected = u"The data: %r." % u"def"
assert_true(expected in response_content)
```

Reply:
Ah that's a great point, thanks Ned.